interface/wx/textctrl.h

```diff
@@ -803,10 +830,25 @@ class wxTextAttr
     void SetFontStyle(wxFontStyle fontStyle);
 
     /**
-        Sets the font underlining.
+        Sets the font underlining (solid line, using text colour).
     */
     void SetFontUnderlined(bool underlined);
 
+    /**
+        Sets the font underlining with a wxTextAttrUnderlineType and wxColour.
+        Specifying wxNullColour will use the text colour.
+
+        @note On wxMSW, wxTEXT_ATTR_UNDERLINE_DOUBLE is shown as wxTEXT_ATTR_UNDERLINE_SOLID. There is only a limited number of colours supported,
```

Comment:
This is trivial and I can do it myself when applying, but this line (and other ones in this comment) should be wrapped.